scipy/sparse/tests/test_coo.py

```diff
@@ -171,7 +292,7 @@ def test_1d_toformats():
         assert_equal(f().toarray(), res.toarray())
 
 
-@pytest.mark.parametrize('arg', [1, 2, 4, 5, 8])
+@pytest.mark.parametrize('arg', [1, 2, 4, 5, 8,])
```

Comment:
yeah, this might be my fault for not being more specific in an earlier review. [PEP8 says](https://peps.python.org/pep-0008/#when-to-use-trailing-commas):

> When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line. However it does not make sense to have a trailing comma on the same line as the closing delimiter (except in the above case of singleton tuples):

so they should be avoided for lists that fit on a single line